apps/rush-lib/etc/examples/Sample_CustomCommandLine.json

```diff
@@ -52,14 +54,14 @@
     "--ship": {
       "optionType": "flag",
       "description": "Perform a production build, including minification and localization steps",
-      "supportedCommands": [ "build", "rebuild" ],
+      "associatedCommands": [ "build", "rebuild", "check-copyright" ],
       "shortName": "-s"
     },
 
     "--minimal": {
```

Comment:
>minimal [](start = 7, length = 7)

You can probably delete this.  It's already in the legacy example, and it's not illustrating anything new here.